src/plugins/discover/public/application/components/sidebar/discover_field.tsx

```diff
@@ -100,6 +101,8 @@ export function DiscoverField({
     }
   );
 
+  const [infoIsOpen, setOpen] = useState(false);
```

Comment:
on line 55 .. `onShowDetails` is no longer used and can be removed, also the the test in `discover_field.test.tsx` on line 108, one test failure less 🎉